pinot-segment-local/src/main/java/org/apache/pinot/segment/local/aggregator/AvgValueAggregator.java

```diff
@@ -26,10 +26,11 @@
 
 public class AvgValueAggregator implements ValueAggregator<Object, AvgPair> {
   public static final DataType AGGREGATED_VALUE_TYPE = DataType.BYTES;
+  public static final AggregationFunctionType AGGREGATION_FUNCTION_TYPE = AggregationFunctionType.AVG;
```

Comment:
(minor) Seems no longer needed, consider reverting them (these types are actually self-explained, so maybe not worth defining as constant)

Reply:
Reverted in https://github.com/apache/pinot/pull/12164/commits/e11de58e72f2fb63f93a11eb0b5f6b19d7cae73d